FirebaseAppDistribution/Sources/FIRSceneHelper.m

```diff
@@ -0,0 +1,75 @@
+
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "FIRSceneHelper.h"
+#import <UIKit/UIKit.h>
+
+API_AVAILABLE(ios(13.0))
+typedef void (^FindSceneCompletionBlock)(UIWindowScene *scene);
```

Comment:
Can we add a `FIR` prefix to the `FindSceneCompletionBlock` name to avoid potential symbol collisions?

Reply:
Yes, agree.